commit 6506fdd08afae77bafb001f8fc6002a877b9af66
Author: Matthias Clasen <mclasen@redhat.com>
Date:   Tue Sep 13 23:55:16 2011 -0400

    color: Fix spawning of gcm-viewer
    
    argv arrays passed to g_spawn need to be NULL-terminated.

diff --git a/panels/color/cc-color-panel.c b/panels/color/cc-color-panel.c
index 47f1429..f47c6dc 100644
--- a/panels/color/cc-color-panel.c
+++ b/panels/color/cc-color-panel.c
@@ -573,6 +573,7 @@ gcm_prefs_profile_view_cb (GtkWidget *widget, CcColorPanel *prefs)
   g_ptr_array_add (argv, g_strdup (cd_profile_get_id (profile)));
   g_ptr_array_add (argv, g_strdup ("--parent-window"));
   g_ptr_array_add (argv, g_strdup_printf ("%i", xid));
+  g_ptr_array_add (argv, NULL);
   ret = g_spawn_async (NULL, (gchar**) argv->pdata, NULL, 0,
                        NULL, NULL, NULL, &error);
   if (!ret)
